kroxylicious-kms-provider-aws-kms-test-support/src/main/java/io/kroxylicious/kms/provider/aws/kms/AbstractAwsKmsTestKmsFacade.java

```diff
@@ -44,6 +43,15 @@
 public abstract class AbstractAwsKmsTestKmsFacade implements TestKmsFacade<Config, String, AwsKmsEdek> {
     private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
     private static final int MINIMUM_ALLOWED_EXPIRY_DAYS = 7;
+
+    private static final String TRENT_SERVICE_DESCRIBE_KEY = "TrentService.DescribeKey";
```

Comment:
Niggle: We should strive to keep the PRs as small as possible, so avoid inconsequential changes like this one.   Reviewer time is valuable.

Reply:
Yeah sorry for that, changing my original plans become a more difficult class to review.